### PR TITLE
Converted add item page to view convention

### PR DESCRIPTION
**Pull request by Manan-dev**
# Description

Converted add item page to view convention

## Relevant issue(s)

#111 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have assigned reviewers to this PR

### CODE REVIEW

File: src/views/addItemView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        console.error('Failed to add item');
+      }
+    } catch (error) {
+      console.error('Error:', error);
+    }
+  };
+
+  return (
+    <>
+      <TopBar />
```
Comment (alay10) [inline src/views/addItemView/index.tsx:51]: Go ahead and move this element into the server component `app/tiem/add/page`

**Review by alay10 — CHANGES_REQUESTED** [review]

File: src/views/addItemView/index.tsx (hunk 1 above)
Comment (Manan-dev) [reply src/views/addItemView/index.tsx:51]: Done

**Comment by AlanKha** [comment]
Already kinda reviewed this from last PR. LGTM

**Review by AlanKha — APPROVED** [review]

**Review by alay10 — APPROVED** [review]
Nice Job!

**Pull request merged**